client/components/thank-you-v2/style.scss

```diff
@@ -14,4 +14,9 @@
 		max-width: 650px;
 		width: 100%;
 	}
+
+	.components-button {
+		border-radius: 4px;
+		padding: 10px 24px;
+	}
```

Comment:
Nice 👍 I overrode this in #86409 for the header (see https://github.com/Automattic/wp-calypso/pull/86409#discussion_r1452473414). I'll update that PR to account for this incoming change